Frameworks/UIKit/UIButton.mm

```diff
@@ -850,19 +794,17 @@ - (UIColor*)currentTitleColor {
 }
 
 /**
- @Status Caveat
- @Notes Returns a mock UILabel that proxies some common properties and selectors to the underlying TextBlock
+ @Status Interoperable
 */
 - (UILabel*)titleLabel {
-    return (UILabel*)[[_proxyLabel retain] autorelease];
+    return [[_textLabel retain] autorelease];
```

Comment:
What is stopping us from making UIButton ARC-enabled like the other XAML-backed controls?